src/wallet/wallet.cpp

```diff
@@ -4320,9 +4320,6 @@ bool CWallet::CreateMultipleZerocoinSpendTransaction(std::string &thirdPartyaddr
                 }
             }
 
-            txHash = wtxNew.GetHash();
-            LogPrintf("wtxNew.txHash:%s\n", txHash.ToString());
-
             // Embed the constructed transaction data in wtxNew.
             *static_cast<CTransaction *>(&wtxNew) = CTransaction(txNew);
```

Comment:
This is really bad practice even if it works. Additional problem is in bitcoin 0.14+ `CWalletTx` and `CTransaction` classes are unrelated.

Reply:
The problems is this is a code from original `CreateTransaction` function which also do the same like this. If we want to fix this we may need to refactor `CWalletTx`.